### ISSUE TITLE
Some log message in json log format are invalid json

### ISSUE THREAD

**Issue by aensidhe** [tarantool/tarantool#3321]
Version: 1.9.0-47-gfabbcfa68

````
{"time": "2018-04-05T14:10:24.093+0000", "level": "CRIT", "message": "entering the event loop", "pid": 1 , "cord_name": "main", , "file": "\/usr\/src\/tarantool\/src\/main.cc", "line": 765}
````

After "main" there are two commas

**Comment by aensidhe** [comment]
````
{"time": "2018-04-06T09:32:51.003+0000", "level": "SYSERROR", "message": "SystemError read(14887), called on fd 24, aka 172.17.0.17:3301", "error": "Operation timed out", "pid": 1 , "cord_name": "iproto", , "file": "\/usr\/src\/tarantool\/src\/sio.cc", "line": 278}
````

**Comment by Gerold103** [comment]
Fixed in 38b2a29ff8bc1041562fb9b48b5f3e7d1ef294b2
